src/scripttabs.cpp

```diff
@@ -108,7 +108,7 @@ TYPE_SYMBOL asTypeTable[] =
 	{ "",	(INTERP_TYPE)ST_POINTER_STRUCTSTAT, AT_SIMPLE, NULL, NULL },	//for NULLSTRUCTURESTAT
 
 	/* This final entry marks the end of the type list */
-	{ "END OF TYPE LIST", (INTERP_TYPE)NULL,		AT_SIMPLE,	NULL,	NULL },
+	{ "END OF TYPE LIST", (INTERP_TYPE)0,		AT_SIMPLE,	NULL,	NULL },
```

Comment:
> Having a lot of variables of static storage duration with complex constructors will significantly affect startup time (those are all execute _before main). That's why I like to avoid things like that.

Significantly? No. I don't believe it will ever be noticeable in practice.

> Something like iterator_range (both from ranges::v3 and boost)

This is overcomplication for no real gain.


Anyway, please merge the fix if it's OK.